workspaces/liferay-learn-workspace/client-extensions/liferay-learn-dxp-importer/app.sh

```diff
@@ -63,6 +63,40 @@ if [ -z "$SKIP_UPDATE_EXAMPLES" ] ; then
 	send_slack_message "update_examples.sh finished with return code $UPDATE_EXAMPLES_RC. $ERROR_COUNT entries in error log file."
 fi
 
+if [ -z "$SKIP_GENERATE_ZIPS" ] ; then
+	cd $REPO_FOLDER/docs || exit
+
+	for docs_dir_name in $(find . -maxdepth 4 -mindepth 4 -type f -name "contents.rst" -printf "%h\n" )
+	do
+		language=$(echo "${docs_dir_name}" | cut -f4 -d'/')
+		product=$(echo "${docs_dir_name}" | cut -f2 -d'/')
+		version=$(echo "${docs_dir_name}" | cut -f3 -d'/')
+
+		product_version_language_dir_name=$(echo "${product}"/"${version}"/"${language}")
+
+		for zip_dir_name in $(find "${product_version_language_dir_name}" -name *.zip -type d)
+		do
+			pushd "${zip_dir_name}"
+
+			zip_file_name=$(basename "${zip_dir_name}")
+
+			7z a ${zip_file_name} ../${zip_file_name}\
+
+			7z rn ${zip_file_name} ${zip_file_name} ${zip_file_name%.*}
+
+			popd
+
+			output_dir_name=$(dirname "/public_html/${zip_dir_name}")
+			output_dir_name=$(dirname "${output_dir_name}")
+			output_dir_name=$(dirname "${output_dir_name}")
```

Comment:
is there some reason to repeat this?

Reply:
So each run of this line is basically popping off the last directory, it's the way that they clean up the directory structure to output to. When matching it up directory structure on the new service with the existing urls, I didn't see any reason to change from this logic.

Reply:
I see. I guess that's just as good as something like ```dirname $( dirname $(dirname "/public_html/${ZIP_DIR_NAME}"))
```